config/file_config.go

```diff
@@ -119,10 +119,21 @@ type HoneycombLoggerConfig struct {
 	APIHost           string `yaml:"APIHost" default:"https://api.honeycomb.io"`
```

Comment:
Not in scope for this PR, but accessor functions for all fields on `HoneycomLoggerConfig` would be nice.

Reply:
I struggle with this; every accessor adds one form of tech debt to save another form of tech debt. In particular, when I broke up the config into subsections, I deliberately did not add accessors for everything. 

I don't feel strongly against it, I just wish there was a better way (JS/java properties would be nice but Go doesn't give us those).